guardian/support-frontend
# Move selected countries into Europe region

## What does this change?
We have an RFC from Marketing to move the following countries from the `International` region to the `EURCountries` region:

+ Azerbaijan - AZ
+ Armenia - AM
+ Georgia - GE
+ Belarus - BY
+ Moldova - MD
+ Ukraine - UA
+ North Macedonia - MK

## How to test
Review a Banner or Epic message, targeted at Europe, in RRCP Web preview. Bu setting the user location to one of the above countries, the message should display copy adapted for that region - eg: amounts should display using the `€` symbol rather than the `$` symbol.

Then click the message CTA button to navigate to the Supporter+ landing page. The user should see the amounts card values priced in Europe, and the currency selector at the top of the page should be set to the Europe region.

## Screenshots
None - this is an entirely internal change which should not affect the layout of the Supporter+ landing page
